libfwupd/fwupd-device.c

```diff
@@ -74,6 +74,7 @@ enum {
 	PROP_PROTOCOL,
 	PROP_STATUS,
 	PROP_PARENT,
+	PROP_UPDATE_STATE,
```

Comment:
With this type of change I almost need to wonder if we really need to be using the device flag for needs activation.  Should we deprecate it and become an update state?